### PR TITLE
code-push-cli: implement “outputDir” parameter

### PR DESCRIPTION
**Pull request by sergey-akhalkov**
“outputDir” parameter is used to specify the path where bundle and sourcemap should be written.

Relate #376

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -554,6 +555,12 @@ This specifies the relative path to where the generated JS bundle's sourcemap fi
 
 *NOTE: This parameter can be set using either --sourcemapOutput or -s*
 
+#### Output directory parameter
+
+This specifies the relative path to where the assets, JS bundle and sourcemap files should be written. If left unspecified, assets, JS bundle and sourcemap will copied to /tmp/CodePush folder by default.
```
Comment (silhouettes) [inline cli/README.md]: Suggested modifications:
- will be copied
- to the `/tmp/CodePush` folder
- Could you put back-ticks around the directory path?

**Comment by sergey-akhalkov** [comment]
Fixed it, thanks.

**Comment by cooperka** [comment]
@sergey-akhalkov awesome! Can this also be added to the docs online at https://microsoft.github.io/code-push/docs/cli.html#releasing-updates-react-native? Should I open a new issue for this?

**Comment by cooperka** [comment]
This also seems to clobber the entire directory, it deleted everything previously under `ios` when outputting there. Maybe a warning about this would be nice :P

**Comment by sergey-akhalkov** [comment]
Hi @cooperka, thank you for reaching us. No need to create new issue, there is a script that should sync up docs from github with CodePush site, so I'll ping guys who can help us with this.

>This also seems to clobber the entire directory, it deleted everything previously under ios when outputting there. Maybe a warning about this would be nice

Got it, should we add a `note` to the docs that will pay attention on such behavior?

**Comment by cooperka** [comment]
@sergey-akhalkov a note in the docs would be perfect. Thanks!

**Comment by sergey-akhalkov** [comment]
@cooperka, I've just published https://github.com/Microsoft/code-push/pull/467, please let me know if it looks good to you.

**Pull request merged**